### PR TITLE
Update skip_if_only_changed settings for shared resource csi driver

### PR DESCRIPTION
**Pull request by coreydaley**
Add skip_if_only_changed settings to newly created jobs for shared
resource csi driver

### CODE REVIEW

File: ci-operator/jobs/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10-presubmits.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 presubmits:
   openshift/csi-driver-shared-resource-operator:
   - agent: kubernetes
-    always_run: true
+    always_run: false
```
Comment (gabemontero) [inline ci-operator/jobs/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10-presubmits.yaml:4]: is this needed for the skip thing ?

Ideally the semantic is "aways run but skip if only changed ..."

Comment (coreydaley) [reply ci-operator/jobs/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10-presubmits.yaml:4]: Yes, it is needed.
That change was done when running "make update" due to the skip_if_only_changed in the configuration files.

Comment (gabemontero) [reply ci-operator/jobs/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10-presubmits.yaml:4]: ok thanks for that confirmation

and we  get the "aways run but skip if only changed ..." semantic, right ?

Comment (coreydaley) [reply ci-operator/jobs/openshift/csi-driver-shared-resource-operator/openshift-csi-driver-shared-resource-operator-release-4.10-presubmits.yaml:4]: correct

**Pull request merged**